CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 ### Added
+### Fixed
+### Changed
+
+## 1.13.0
```

Comment:
Need a release date here?

Also, do we want to wait for #371 to get in for this release? looks like it's about to merge.

Reply:
nah, don't wait

Reply:
updated the release date but I don't think we should wait.